lists/ua.csv

```diff
@@ -1,887 +1,941 @@
-http://www.who.int/topics/family_planning/ru/,XED,Sex Education,2014-04-15,citizenlab,
-http://www.wikia.com/wikia,HOST,Hosting and Blogging Platforms,2014-04-15,citizenlab,
-http://www.winenews.ru,ALDR,Alcohol & Drugs,2014-04-15,citizenlab,
-http://www.womnet.ru,HUMR,Human Rights Issues,2014-04-15,citizenlab,
-http://www.worldmeets.us,NEWS,News Media,2014-04-15,citizenlab,
-http://www.wsws.org,NEWS,News Media,2014-04-15,citizenlab,
-http://www.wsws.org/ru/1999/dez1999/heal-d03.shtml,PUBH,Public Health,2014-04-15,citizenlab,
-http://www.wuor.ru,HUMR,Human Rights Issues,2014-04-15,citizenlab,
-http://www.wur.newmail.ru,HUMR,Human Rights Issues,2014-04-15,citizenlab,
-http://www.xakep.ru,HACK,Hacking Tools,2014-04-15,citizenlab,
-http://www.yabloko.ru,POLR,Political Criticism,2014-04-15,citizenlab,
-http://www.yandex.ru,NEWS,News Media,2014-04-15,citizenlab,
-http://www.yapfiles.ru,NEWS,News Media,2014-04-15,citizenlab,
-http://www.yaplakal.com,NEWS,News Media,2014-04-15,citizenlab,
-http://www.yhrm.hrworld.ru,HUMR,Human Rights Issues,2014-04-15,citizenlab,
-http://www.yhrm.org,POLR,Political Criticism,2014-04-15,citizenlab,
-http://www.youtube.com/watch?v=ytholtbgyxy,POLR,Political Criticism,2014-04-15,citizenlab,
-http://www.zaprava.ru,POLR,Political Criticism,2014-04-15,citizenlab,
-http://www.zdn.su,PUBH,Public Health,2014-04-15,citizenlab,
-http://www.zerkalo-nedeli.com,NEWS,News Media,2014-04-15,citizenlab,
-http://www.znopr.ru,PUBH,Public Health,2014-04-15,citizenlab,
-http://wwwoman.ru,XED,Sex Education,2014-04-15,citizenlab,
-http://wwwoman.ru/s.php/755.htm,XED,Sex Education,2014-04-15,citizenlab,
-http://x-dvd.com.ua,PORN,Pornography,2014-04-15,citizenlab,
-http://x-video.com.ua,PORN,Pornography,2014-04-15,citizenlab,
-http://xmob.kiev.ua,PORN,Pornography,2014-04-15,citizenlab,
-http://xn--80aaifmgl1achx.xn--p1ai,GMB,Gambling,2014-04-15,citizenlab,
-http://xs.gay.ru,LGBT,LGBT,2014-04-15,citizenlab,
-http://xterror.narod.ru,HACK,Hacking Tools,2014-04-15,citizenlab,
-http://xvideos.com,NEWS,News Media,2014-04-15,citizenlab,
-http://yadumayu.ru,NEWS,News Media,2014-04-15,citizenlab,
-http://yashin.livejournal.com,POLR,Political Criticism,2014-04-15,citizenlab,
-http://yesche.biz,ALDR,Alcohol & Drugs,2014-04-15,citizenlab,
-http://yhrm.hrworld.ru,HUMR,Human Rights Issues,2014-04-15,citizenlab,
-http://youtube.com,NEWS,News Media,2014-04-15,citizenlab,
-http://yury108.blogspot.com,NEWS,News Media,2014-04-15,citizenlab,
-http://zabormozga.com,ALDR,Alcohol & Drugs,2014-04-15,citizenlab,
-http://zaukraine.com/blog,NEWS,News Media,2014-04-15,citizenlab,
-http://zhurnal.lib.ru,MISC,Miscelaneous content,2014-04-15,citizenlab,
-http://zn.ua,NEWS,News Media,2014-04-15,citizenlab,
-http://znaesh.net,SRCH,Search Engines,2014-04-15,citizenlab,
+url,category_code,category_description,date_added,source,notes,,,,,,,
+http://24tv.ua,NEWS,News Media,2014-04-15,citizenlab,,,,,,,,
+http://2ch.hk,PORN,Pornography,2014-04-15,citizenlab,,,,,,,,
```

Comment:
Same story here, but it's clear here -- there are some buggy empty columns in the end of each line.

Heh, forms, databases... Now I clearly see the reason for them ;-)

@hellais look, here is an argument :-)

Reply:
Looks like this is something the lint script should detect

Reply:
It did detect it actually :P